modules/apps/site/site-cms-site-initializer/src/main/resources/META-INF/resources/js/main/spaces/NewSpace.tsx

```diff
@@ -50,6 +50,9 @@ const NewSpace = ({baseRedirectUrl}: NewSpaceProps) => {
 					'spaces-are-essential-for-organizing-defining-and-managing-your-content-and-files'
 				)}
 				linkLabel={Liferay.Language.get('learn-more-about-spaces')}
+
+				// TODO: update this link
```

Comment:
Please remove comment ;-)

Reply:
I left just for us to not forget to update this link, but since we have a new story for that, I'm removing it...